FreightLine.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include <iostream>
+#include <vector>
+#include "LeakedObjectDetector.h"
+
+#include "Wrappers.h"
+
+struct FreightLine
+{
+    TrainWrapper train1 = TrainWrapper(new FreightTrain(4,20,"Fred"));
+    TrainWrapper train2 = TrainWrapper(new FreightTrain(2,25,"Bob"));
+    TrainWrapper train3 = TrainWrapper(new FreightTrain());
+    //TrainWrapper train3(new FreightTrain()); // not sure I understand why I can't declare and initialize wrapper class member variables like this
```

Comment:
see my comment in slack regarding why you can't wrap member variables
- [ ] revise, remove the wrapper usage